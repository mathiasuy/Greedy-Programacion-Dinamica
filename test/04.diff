1,2c1,13
< 1>Volumen: 484
< 2>Fin.
---
> ==4615== Conditional jump or move depends on uninitialised value(s)
> ==4615==    at 0x1091FE: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4615==    by 0x109679: main (in /mnt/c/p3/tareap3ant/principal)
> ==4615== 
> ==4615== Invalid read of size 1
> ==4615==    at 0x1091F9: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4615==    by 0x109679: main (in /mnt/c/p3/tareap3ant/principal)
> ==4615==  Address 0x5b7e461 is 0 bytes after a block of size 1 alloc'd
> ==4615==    at 0x4C3017F: operator new(unsigned long) (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
> ==4615==    by 0x109ED4: max_volumen(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4615==    by 0x109653: main (in /mnt/c/p3/tareap3ant/principal)
> ==4615== 
> 1>
\ No newline at end of file
