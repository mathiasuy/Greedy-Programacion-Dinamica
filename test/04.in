max_volumen_intervalos 
30
1-15-18
2-8-35
15-30-46
27-35-57
20-33-12
9-15-10
10-22-98
25-40-24
45-63-1
50-57-37
4-35-88
10-42-54
30-58-23
61-64-19
63-75-15
59-63-70
63-70-1
43-75-86
60-78-72
67-70-65
8-15-43
30-33-16
39-55-80
25-43-78
38-40-75
48-77-28
65-77-45
74-78-47
34-42-50
50-68-21
Fin
