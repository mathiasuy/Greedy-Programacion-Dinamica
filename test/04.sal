==4392== error calling PR_SET_PTRACER, vgdb might block
==4392== Conditional jump or move depends on uninitialised value(s)
==4392==    at 0x1090FE: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3/principal)
==4392==    by 0x109579: main (in /mnt/c/p3/tareap3/principal)
==4392== 
==4392== Invalid read of size 1
==4392==    at 0x1090F9: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3/principal)
==4392==    by 0x109579: main (in /mnt/c/p3/tareap3/principal)
==4392==  Address 0x5b7e461 is 0 bytes after a block of size 1 alloc'd
==4392==    at 0x4C3017F: operator new(unsigned long) (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==4392==    by 0x109CA9: max_volumen(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3/principal)
==4392==    by 0x109553: main (in /mnt/c/p3/tareap3/principal)
==4392== 
1>