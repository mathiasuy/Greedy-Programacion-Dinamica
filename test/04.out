1>Volumen: 484
2>Fin.
